### DISCUSSION TITLE
G7 not pairing to new phone

### DISCUSSION THREAD

**Discussion by dbfreud** [NightscoutFoundation/xDrip#3110] [Q&A]
I just switched to a Samsung S22 and can not get it to pair. I checked the app settings recommended for G7 were imported correctly from my S20

![Screenshot_20231002_143642_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/b179e22d-3177-41fc-a5de-dde7b8c3b27c)

**Comment by Navid200** [comment]
Please post a screenshot of the G5/G6/G7 status page: https://navid200.github.io/xDrip/docs/StatusG5G6.html

**Reply by dbfreud** [reply]
![Screenshot_20231002_153935_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/9334ee69-c408-4e06-ba1c-fc71530ca724)

![Screenshot_20231002_154008_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/3ba90824-36b0-4c13-9505-fc8e37bea3a7)

**Reply by Navid200** [reply]
Can you scan the QR code on the new phone again?
https://navid200.github.io/xDrip/docs/Dexcom/G7.html

**Reply by dbfreud** [reply]
Do you mean this?

![Screenshot_20231002_160248_Chrome.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/8df6ff01-26a9-49d7-bc3b-eecc84f55ed6)

**Comment by dbfreud** [comment]
that did it thanks very much.  next I need to get it back on my wearable

**Reply by dbfreud** [reply]
I'm unfortunately  having the same problem as the first time I set up xdrip. It stopped working when I changed the settings to enable wearable ![Screenshot_20231002_184604_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/780cb018-2bd3-4578-9b8f-b4fc70f1ef7b)

![Screenshot_20231002_184939_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/bbf7ccb5-d38d-4711-9059-057b867cf8c3)

**Reply by Navid200** [reply]
We don't have an active developer supporting WearOS.

**Comment by dbfreud** [comment]
Nothing is working for me. The app stopped functioning on the phone as it
did last time. I have started a second transmitter on a different device
using the dexcom app so and will keep trying to get xdrip going but will
abandon if I can't get it to work

**Reply by dbfreud** [reply]
Still not connecting ![Screenshot_20231003_144820_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/b073caf4-d688-4295-81f4-5defee74def8)

**Reply by dbfreud** [reply]
What does "phone service state not running" mean?

**Reply by Navid200** [reply]
We already went over everything and you said it was working until you enabled Wear.

I already told you that there is no one who supports wear.  
Your issue isn't really G7.  It is wear.

If you are hoping someone would help you with wear, that is possible.
If you are hoping someone will help you with G7, there is nothing to help you with.  I need to know what is not  working with respect to G7 and I will track it.  But, if you mix the two issues, I will have to stay out even if you keep the title focused on G7.

**Comment by dbfreud** [comment]
What screenshots do you need to know what's not working with respect to G7?

**Reply by Navid200** [reply]
None because G7 is working fine.
I don't know how else to explain that Wear is your problem.

**Comment by dbfreud** [comment]
G7 isn't working, it's not picking up readings.  I am not asking about
Wear, I'm asking for support in getting my G7 device to function again
using xdrip

**Reply by Navid200** [reply]
So, when you made this post, it wasn't working?
![Screenshot 2023-10-03 152205](https://github.com/NightscoutFoundation/xDrip/assets/51497406/c199314b-2f18-48d8-b3e8-1d57b85eaf3b)

**Comment by dbfreud** [comment]
Of course it was working.  After it began working I then attempted to
connect to Wear (apologies for including in this string but it's the only
way to answer your question).  To do so, I updated the settings in Smart
Watch Features as oultined.  I also reset my wear device to reconnect with
my new phone but xdrip didn't transfer over and I haven't been able to
re-install it.

**Reply by Navid200** [reply]
The reason it stopped working was that you enabled Wear.

G7 is new.  No one has updated Wear OS recently in xDrip.  Certainly not after G7 capability was added to xDrip.  So, I am not surprised if you have difficulty using it WearOS.  You are not the only one.

**Comment by dbfreud** [comment]
xDrip worked perfectly on my last devices, phone and wear both, so I assume
I did something wrong attempting to set up again. Regardless, I was just
trying to get xDrip working on the phone, If I can't get itworking at all
I'll need to abandon it.
